### PR TITLE
Remove Timeline JS

### PR DESCRIPTION
**Pull request by brandonhundt**
--- its been cut from the project.

### CODE REVIEW

File: pages/garages.html [hunk 1]
```diff
@@ -31,8 +31,8 @@
 recommendation-3-link: https://www.radiotopia.fm/podcasts/the-memory-palace
 partner-link: dovetail
 partner-title: PRX Technology Lets Stations Take Control
-creator-link: keyshawn
-creator-title: Keyshawn Solves it
+creator-link: nairobi-popup
+creator-title: Nairobi Pop-up Garage
```
Comment (rpeterman-gp) [inline pages/garages.html]: This fix used a different title in PR #20. Either this PR needs to not include the fix, or gets merge before #20.

**Review by rpeterman-gp — CHANGES_REQUESTED** [review]
The fix in the garage page might override the fix in the other PR.

File: pages/garages.html (hunk 1 above)
Comment (rpeterman-gp) [reply pages/garages.html]: Probably address this when handling the merge conflicts.

Comment (brandonhundt) [reply pages/garages.html]: Yes, resolving the merge conflicts fixed this.

**Comment by brandonhundt** [comment]
@rpeterman-gp Ready for you to look at again. I resolved the issues in the merge conflict. Thanks.

**Review by rpeterman-gp — APPROVED** [review]

**Pull request merged**